apollo-backend/src/main/java/io/logz/apollo/services/DeploymentService.java

```diff
@@ -39,6 +39,10 @@ public void logDeploymentDescription(Deployment deployment) {
             MDC.put("markers", String.format("service-name:%s", service.getName()));
             MDC.put("env", env.getAvailability());
             MDC.put("region", env.getGeoRegion());
+            MDC.put("service", service.getName());
+            MDC.put("commit", deployableVersion.getGitCommitSha());
+            MDC.put("committer", deployableVersion.getCommitterName());
+            logger.info("Apollo deployed the commit message = {} triggered by {}", deployableVersion.getCommitMessage(), deployment.getUserEmail());
```

Comment:
@mikeskali you can use the existing log message (with HTML tag) to get statistics.
Or you prefer to have a separate message?

Reply:
I am ok with using the html log and add fields to it. not sure why we print html though.